tensorflow/core/common_runtime/next_pluggable_device/next_pluggable_device.cc

```diff
@@ -52,15 +52,19 @@ NextPluggableDevice::NextPluggableDevice(const SessionOptions& session_options,
                                                      options.device_ordinal)),
       device_ordinal_(options.device_ordinal),
       compilation_device_type_(options.compilation_device_name) {
-  allocator_ = std::make_unique<NextPluggableDeviceAllocator>(device_ordinal_);
   if (absl::GetFlag(FLAGS_next_pluggable_device_use_pjrt)) {
```

Comment:
Do you mind adding a new flag FLAGS_next_pluggable_device_use_pjrt_allocator to guard this behavior?

Reply:
@jyingl3 done, please have a review. thanks very much.

Reply:
Approved the change. Thank you so much!